source/jormungandr/jormungandr/street_network/taxi.py

```diff
@@ -133,10 +133,10 @@ def _add_additional_section_after_first_section_taxi(self, journey, pt_object, a
         # Because Jormun does not do it afterwards
         journey.sections[-1].destination.CopyFrom(pt_object)
 
-        self._update_journey(journey, additional_section)
+        self._update_journey(journey, additional_section, 1)
```

Comment:
how about 
```suggestion
        self._update_journey(journey, additional_section, len(journey.sections))
```

Reply:
I'll try that but I think the insert isn't working properly on the CI, failling the test.